style.css

```diff
@@ -540,3 +546,238 @@ span {
+  body {
+    width: 100%;
+    overflow-x: auto;
+  }
+
+  .contact textarea {
+    font-size: 16px;
+    border-radius: 0;
+    width: 319px;
+    height: 100px;
+  }
+
+  .ending-bar { display: none; }
+
+  .wrap .contact button {
+    align-self: center;
+    padding: 10px;
+  }
+
+  .contact {
+    justify-content: center;
+    align-items: center;
+  }
+
+  .wrap .contact-para { margin-bottom: 20px; }
+
+  .contact-title { margin-bottom: 15px; }
+
+  #navigator_bar {
+    margin-top: 0;
+    margin-right: 5%;
+    margin-left: 10.76%;
+  }
+
+  header #navigator_bar .logo {
+    margin-right: 30%;
+    font-family: poppins, sans-serif;
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  #navigator_bar nav {
+    display: none;
+  }
+
+  #navigator_bar .internal-link {
+    display: flex;
+    justify-content: space-between;
+    min-width: 190px;
+    margin-top: 30px;
+    line-height: 20px;
+  }
+
+  #navigator_bar .internal-link li {
+    list-style: none;
+  }
+
+  #navigator_bar .internal-link li a {
+    text-decoration: none;
+    font-family: poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .starting-page {
+    height: 100vh;
+    background-image: url("assets/Group\ 58.png");
+    background-repeat: no-repeat;
+    background-size: 90%;
+  }
+
+  #section-presentation {
+    height: 916px;
+    margin-left: 18%;
+    margin-right: 18%;
+    margin-top: 200px;
+    justify-content: stretch;
+    background-image: none;
+  }
+
+  #my_work {
+    margin-top: 100px;
+    display: grid;
+    grid-template-rows: repeat(18, 100vh);
+  }
+
+  .general {
+    grid-column: 2 / 1 span;
+    align-self: center;
+  }
+
+  .project {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    width: 80%;
+    height: 90%;
+    margin-left: 10%;
+    grid-column-gap: 20px;
+    margin-bottom: 100px;
+  }
+
+  #general_inf {
+    grid-column: 2 / 1 span;
+  }
+
+  .project footer .external_link {
+    margin-left: 0;
+    width: 18%;
+    padding: 2% 0;
+  }
+
+  .project_image {
+    grid-column: 1 / 1 span;
+    width: 95%;
+    justify-self: center;
+    margin-left: 16px;
+    align-self: center;
+  }
+
+  #about-myself {
+    display: grid;
+    grid-template-columns: 40% 60%;
+    margin: 0 8% 10%;
+    grid-column-gap: 5%;
+  }
+
+  .myself-abt {
+    grid-column: 1 / span 1;
+    margin-right: 0;
+  }
+
+  .myself-abt h1 { margin-bottom: 0; }
+
+  #resume-link h2 { margin-bottom: 0; }
+
+  #resume-link ul { margin-top: 8px; }
+
+  #presentation { margin-bottom: 0; }
+
+  #my_skills {
+    grid-column: 2 / span 1;
+    margin-top: 156px;
+    margin-left: 0;
+    width: 100%;
+    height: 300px;
+    justify-content: space-around;
+  }
+
+  .spec_name {
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  #programmes {
+    flex-direction: row;
+    justify-content: space-between;
+    margin-right: 0;
+  }
```

Comment:
- [OPTIONAL] Kindly make sure the Languages cards are aligned correctly in order to match the Figma design as much as possible.

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/221076823-d9cd7c50-9459-4021-ae8c-9e3fcb59988e.png)

   - ### FIGMA DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/221076931-6374c196-5a8e-4c3f-bbef-88696b2d4be9.png)

Reply:
Thank you for notifying and helping me to solve this. I have resolved the issue